ejviola/mailroom.py

```diff
@@ -1,104 +1,133 @@
 #!/usr/bin/env python3
 
-# Create a list to hold lists of donations
-donordata = []
+# ______________________________________________________________________________________________________________________
+# IMPORT DEPENDENT MODULES
+import pickle
+import sys
+# ______________________________________________________________________________________________________________________
+# DEFINE GLOBAL VARIABLES
+donor_data_file_name = '/home/vagrant/PycharmProjects/git/Py100-2017q1/ejviola/DonorData.pkl'
+# Open donor data, read donor data into a new list of dicts, close donor data file
+try:
+    donor_data_file = open(donor_data_file_name, 'rb')
+    donor_data = pickle.load(donor_data_file)
+    donor_data_file.close()
```

Comment:
You can make these three lines a one liner:
`donor_data = pickle.load(open(donor_data_file_name, 'rb'))`